templates/templates/docs/shared/_docs.html

```diff
@@ -98,6 +98,30 @@ <h3 class="p-side-navigation__heading">{{ nav_group.navlink_text }}</h3>
     </aside>
 
     <main class="col-9">
+      {% if document.headings_map %}
```

Comment:
I don't think you can avoid doing what we discussed and explicitly adding the title above this table of contents. It looks very odd at the top before the heading.

Reply:
Ok, will we keep the `<h1>` in the body of the document?